dist/api_data.py

```diff
@@ -582,6 +582,9 @@ def __ge__(self, other):
             and one that will never checkpoint, it might discard log files before any checkpoint is
             done.)''',
             min='0', max='1024'),
+        Config('update_restore_evict', 'false', r'''
```

Comment:
Hey @kommiharibabu, not quite sure what would be an effective comment here, what do you think would be a suitable comment?